micro-rdk/src/common/board.rs

```diff
@@ -210,6 +226,16 @@ impl Board for FakeBoard {
         Err(BoardError::I2CBusNotFound(name))
     }
 
+    fn add_digital_interrupt_callback(
```

Comment:
I think it would be good to have a doc string comment here? I think there's enough to explain as to why the signature is the way it is and the way developers should call this method